apps/citizen-func/src/functions/GetMessages/__tests__/handler.test.ts

```diff
@@ -275,7 +275,7 @@ const getTaskMock = vi.fn(
 );
 vi.spyOn(redis, "getTask").mockImplementation(getTaskMock);
 
-const redisClientMock = {} as any;
+const redisClientMock = {} as unknown as RedisClientFactory;
```

Comment:
can you replace this `as unknown as RedisClientFactory` with a proper `vi.mock` usage? we should avoid using `as` because is as unsafe as `any`